src/middlewared/middlewared/alert/source/applications.py

```diff
@@ -1,6 +1,21 @@
 from middlewared.alert.base import AlertClass, AlertCategory, AlertLevel, Alert, OneShotAlertClass
 
 
+class FailuresInAppMigrationAlertClass(AlertClass, OneShotAlertClass):
+    deleted_automatically = False
+
+    category = AlertCategory.APPLICATIONS
+    level = AlertLevel.ERROR
+    title = 'Application(s) failed to migrate from kubernetes'
+    text = 'App(s) failed to migrate from dragonfish, please check /var/log/app_migrations.log for more details.'
```

Comment:
You have `App(s)` and `Application(s)`. We should standardize on 1 or the other but not both.